spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Cell.scala

```diff
@@ -32,8 +33,10 @@ import scala.reflect.ClassTag
  *     (because each time step a LSTM return two hiddens `h` and `c` in order,
  *     which have the same size.)
  */
-abstract class Cell[T : ClassTag](val hiddensShape: Array[Int])
-  (implicit ev: TensorNumeric[T])
+abstract class Cell[T : ClassTag](
+  val hiddensShape: Array[Int],
+  val regularizers: Array[Regularizer[T]] = null
```

Comment:
Because the array `regularizers` is set from the outside, if an default empty array is set, maybe it won't be used and overlap